src/java/org/apache/cassandra/db/lifecycle/Tracker.java

```diff
@@ -458,6 +459,8 @@ public Memtable getMemtableFor(OpOrder.Group opGroup, CommitLogPosition commitLo
      */
     public Memtable switchMemtable(boolean truncating, Memtable newMemtable)
     {
+        notifyCreated(newMemtable);
```

Comment:
Should we only notify when `truncating` is false?

Reply:
Just looked up what truncated is. It doesn't seem that we should branch on that boolean.